### ISSUE TITLE
When we start DOSBOX-X 64bit  in an RDP environment, according to emulated CPU speed, every RDP user takes up around 10 percent of processing power without any DOS command being run

### ISSUE THREAD

**Issue by rabhat08** [joncampbell123/dosbox-x#1955]
**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Environment (please complete the following information):**
 - Operating system
 - DOSBox-X release version or commit SHA1 this issue occurred on
 - Used configuration, e.g. ```dosbox-x.conf```

**Additional context**

**Comment by Wengier** [comment]
What RDP environment do you use? Do you have any screenshot showing the data? Thanks!

**Comment by rabhat08** [comment]
Windows 2016 Server.

I have multiple users who need to work in DOS on the same server since the
program is old. RAM is not an issue. To get local type speed they are
setting the "Emulate CPU Speed" to Pentium 166Mhz MMX ". The Processing
power consumed id around 10% per user which is a problem. This is a Core i7
12 core server still there is processing power issue. I have more than 50
users. I can RDP 50 users easily. but when the users open DOSBOX-X and
increase the emulated speed,  in 10 users itself 100 % processing power is
being utilized which is the case of concern. These users were previously
using pentium III CPUs with 512MB RAM. Kindly guide.

**Comment by Wengier** [comment]
DOSBox-X includes a built-in command called DOSIDLE, which can be used to reduce the CPU power. Can you try it and see if it helps?

**Comment by rabhat08** [comment]
didi u receive the 3 attachments sent containing the screenshots. Also where/how do i use the DOSIDLE command?

**Comment by Wengier** [comment]
@rabhat08 I did not see the screenshots yet. I think they needed to be pasted into the reply box directly for them to show up.

On the other hand, the usage of the DOSIDLE command is very easy - just type DOSIDLE in DOSBox-X's DOS shell and that is it.

**Comment by rderooy** [comment]
Also, you may want to try a development build and set ``core=dynamic``. It can have a significant performance impact, as long as your running DOS programs in DOSBox-X and not Win9x which is currently incompatible with the dynamic core.

**Comment by rabhat08** [comment]
This reduced the power. But still does takes 3 to 5% processing power when
executing commands

**Comment by Wengier** [comment]
@rabhat08 That is good. On the other hand, I think it is not possible for DOSBox-X to not take any processing power when executing commands, correct?

**Comment by rabhat08** [comment]
yes. thank u. DOSIDLE, dynamic core and changing "start in" option (to the
folder where the custom conf file is present) in the properties of the
DOSBOX-X shortcut resolved the problem. Thanks a lot everyone. grateful to
u.
